pkg/konfluxgen/integration-test-scenario.template.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: appstudio.redhat.com/v1beta2
 kind: IntegrationTestScenario
 metadata:
-  name: {{{ truncate ( sanitize .ApplicationName ) }}}-enterprise-contract
```

Comment:
as the new ITC do not have the `-enterprise-contract` suffix, we need to delete the old ones, when this merged and got applied

Reply:
it helps to know what's the test about from the name